packages/gdl-frontend/components/Reader/index.js

```diff
@@ -36,39 +37,46 @@ const Reader = ({
   userHasEditAccess
 }: Props) => {
   const isRtlLanguage = !!book.language.isRTL;
-
   return (
-    <Container size="large" gutter={false}>
-      <Backdrop />
-      <Card>
-        <Toolbar
-          book={book}
-          chapter={chapterPointer}
-          userHasEditAccess={userHasEditAccess}
-          onRequestClose={onRequestClose}
-        />
-        {/*
+    <FontSizeContext.Consumer>
+      {/*$FlowFixMe: Ignore flow errors from Context */}
+      {({ state, changeFontSize }) => (
```

Comment:
Since I commented that the Consumer can be used directly in the `<FontSizeComponent>`, you only need the `state` (commented on renaming this to `fontSize` above.